src/hubot-weather.coffee

```diff
@@ -18,11 +19,15 @@ module.exports = (robot) ->
     APIKEY = process.env.HUBOT_OWM_APIKEY or null
     if APIKEY == null
       msg.send "HUBOT_OWM_APIKEY environment varibale is not provided for hubot-weather"
-    msg.http("http://api.openweathermap.org/data/2.5/weather?q=#{msg.match[1]}&units=metric&APPID=#{APIKEY}")
+    UNITS = process.env.HUBOT_WEATHER_UNITS or "metric"
```

Comment:
First, thank you @erikssource for contribution! :) And sorry for delay...

Now about code.

Can we simplify it here?

What about this:

```js

// we define an object of metrics to their representation
units = {metric: "C", imperial: "F"}

UNITS_ENV = process.env.HUBOT_OWM_APIKEY // saving provided key
unitsKey = if units[UNITS_ENV] then UNITS_ENV else "metric" // if it is provided, use it, otherwise "metric"
  
console.log("http://api.openweathermap.org/data/2.5/weather?q=[...]&units=#{unitsKey}&APPID=[...]")
console.log("Forecast for today [...] #{units[unitsKey]}")

```